libgraph/include/katana/GraphTopology.h

```diff
@@ -319,19 +343,14 @@ class KATANA_EXPORT EdgeShuffleTopology : public GraphTopology {
       PropIndexVec&& edge_prop_indices) noexcept
       : Base(std::move(adj_indices), std::move(dests)),
         is_valid_(true),
-        tpose_state_(tpose_todo),
-        edge_sort_state_(edge_sort_todo),
         edge_prop_indices_(std::move(edge_prop_indices)) {
     KATANA_LOG_DEBUG_ASSERT(edge_prop_indices_.size() == num_edges());
+    tpose_state_ = tpose_todo;
+    edge_sort_state_ = edge_sort_todo;
```

Comment:
why the move?

Reply:
Base class variables cannot be initialized in the initializer list of a derived class's constructor. I could either add them to the `GraphTopology` constructor arguments or to put them into the constructor body. I chose the latter because I don't want to allow creating instances of `GraphTopology` with different values for these two variables.

Reply:
Thinking about it some more, pushing these to the base class constructor would be much cleaner.